src/Tools/dotnet-dsrouter/README.md

```diff
@@ -0,0 +1,3 @@
+# dotnet-dsrouter
+
+Please refer to [dotnet-dsrouter doc](../../../documentation/dotnet-dsrouter-instructions.md) for instructions on how to use this tool.
```

Comment:
Is this doc coming in this PR?

Reply:
Will be added later. Initially copy paste, will remove the refer for now.